src/components/ui/Mark.tsx

```diff
@@ -0,0 +1,70 @@
+import classnames, {
+  alignItems,
+  borderColor,
+  borderRadius,
+  borderWidth,
+  boxSizing,
+  cursor,
+  display,
+  fontFamily,
+  fontSize,
+  height,
+  justifyContent,
+  width,
+} from 'classnames/tailwind'
+
+const questionStyles = (size?: Sizes, disabled?: boolean) =>
+  classnames(
+    display('flex'),
+    justifyContent('justify-center'),
+    alignItems('items-center'),
+    fontSize(size === Sizes.Small ? 'text-xs' : 'text-base'),
+    fontFamily('font-primary'),
+    cursor(disabled ? undefined : 'cursor-pointer')
+  )
+
+const borderWrapper = (size?: Sizes) => {
+  return classnames(
+    boxSizing('box-content'),
+    borderRadius('rounded-full'),
+    borderWidth('border'),
+    borderColor('border-current'),
+    width({
```

Comment:
You have a square, this means you can remove `height` or `width` and set `h-auto`/`w-auto`

Reply:
by square I mean that your box has w=h, not that the mark is square :)

Reply:
this didn't work, unfortunately